### DISCUSSION TITLE
cc1101 868mhz module?

### DISCUSSION THREAD

**Discussion by mr-mbs** [arnemauer/Ducobox-ESPEasy-Plugin#119] [Q&A]
Hi Arne,

Ben aan de hand van je plugin en de documentatie zelf alles aan het opzetten. Wemos d1 pro gekoppeld aan de cc1101 868mhz module. Ik vermoed dat ik mijn cc1101 868mhz module kapot heb gesoldeerd. Die module is echt super klein en je kan er niet makkelijk header pins op  zetten. 
Heb jij daar tips voor?
Of weet jij een cc1101 868mhz module ergens te koop die wat groter is of al pins voor gesoldeerd heeft?

Alvast bedankt voor je reactie!

**Comment by arnemauer** [comment]
Wat voor module heb je? De modules die ik gebruik zijn best groot (3 bij 3cm).

**Reply by mr-mbs** [reply]
De module van mij is 2 bij 1,5 cm.
Waar koop je die die jij hebt?

**Reply by arnemauer** [reply]
Ik maak gebruik van deze module:
![cc1101 module](https://user-images.githubusercontent.com/268051/204051098-03455740-b134-40cd-a54e-044850e1df83.jpg)

In de nieuwe gateway zit het CC1101 circuit op de pcb zelf (zonder module).

**Reply by mr-mbs** [reply]
Yes die heb ik ook. Ik vermoed dus dat ik de mijne kapot heb gemaakt. ik krijg volgende melding: Initialisation -> calibration failed. No response from CC1101 or status not idle.

Zou het mogelijk ook lukken met deze module?
https://www.aliexpress.us/item/2251832872452748.html?spm=a2g0o.store_pc_allProduct.8148356.3.2d8045fcptntSZ&pdp_npi=2%40dis%21USD%21US%20%243.80%21US%20%242.74%21%21%21%21%21%400b0a0ae216695590362694991ed7f3%2167494058726%21sh&gatewayAdapt=glo2usa&_randl_shipto=US

Hier zitten al pins op, lekker makkelijk.

**Reply by YoYoost** [reply]
@mr-mbs ik heb exact hetzelfde issue. Ik denk niet dat mijn soldeer-skills voldoende ontwikkeld zijn. Is het met (een vergelijkbare) oplossing als bovenstaand gelukt?

**Reply by arnemauer** [reply]
Die module is 433mhz dus dat gaat niet lukken. Waarmee solderen jullie de module? Met een solderbout met fijne punt zijn deze makkelijk te solderen. Is jullie werkwijze wel goed?

Bout warm genoeg, goede soldeertin, eventueel flux erbij, de pad van de module goed verwarmen, niet het tin op de soldeerbout laten smelten maar op de pad.

**Reply by LRAB1** [reply]
Bovenstaand probleem vandaag opgelopen. Deze discussie lezende ga ik er vanuit dat mijn 868mhz module niet juist is gesoldeerd ook al meet ik op de alle pins wel verbinding naar de D1.
